src/sentry/utils/appleconnect/appstore_connect.py

```diff
@@ -289,15 +289,15 @@ def get_build_info(
             "&limit=200"
             # include related AppStore/PreRelease versions with the response as well as
             # buildBundles which contains metadata on the debug resources (dSYMs)
-            "&include=appStoreVersion,preReleaseVersion,buildBundles"
```

Comment:
nit: why the reordering here?

Reply:
Yeah, it's silly:
* Apple re-orders these in this order in the `self` and `next` links in API responses
* The tests sets up `responses` with the URLs from `self` and `next` from the recorded responses.
* `responses` is smart enough to understand query parameters can be re-ordered when mapping requests to responses, but it does not know these arguments to a parameter are equivalent.

So the simplest thing was to align this with how apple builds the response